src/foam/u2/detail/SectionedDetailPropertyView.js

```diff
@@ -226,7 +226,6 @@ foam.CLASS({
       this
         .addClass(this.myClass())
         .add(this.slot(function(mode, prop, prop$label) {
-          if ( mode === self.DisplayMode.HIDDEN ) return null;
```

Comment:
Won't removing this cause the labels and tooltips of the property to appear?

Reply:
I just tested out adding label and help to a sectioned property that has visibility: 'HIDDEN' set, and nothing from the property is displayed.